src/lib/bootloader/grub2_widgets.rb

```diff
@@ -671,12 +673,9 @@ def label
 
     def help
       _(
-        "<p> By pressing <b>Set as Default</b> you mark the selected section as\n" \
-        "the default. When booting, the boot loader will provide a boot menu and\n" \
-        "wait for the user to select a kernel or OS to boot. If no\n" \
-        "key is pressed before the timeout, the default kernel or OS will\n" \
-        "boot. The order of the sections in the boot loader menu can be changed\n" \
-        "using the <b>Up</b> and <b>Down</b> buttons.</p>\n"
+        "<p><b>Default Boot Section</b> selects boot section to be booted as default.\n" \
+        " If YaST2 does not know yet naming of sections, it will be empty and default \n" \
```

Comment:
Hm, this is hard to understand, could you rephrase it? I do not know how to write that better... :-/